Pass path as string to mkpath to avoid:
> mkpath: 'name' must be a string (got PosixPath

Author: Alexander Grund (TU Dresden)

diff --git a/setup.py b/setup.py
index 84f1db16..482d072a 100644
--- a/setup.py
+++ b/setup.py
@@ -138,7 +138,7 @@ def build_extension(self, ext: Extension) -> None:  # noqa: C901
             with unset_python_path():
                 self.spawn([cmake, '--version'])  # cmake in the parent virtual environment
 
-        self.mkpath(build_temp)
+        self.mkpath(str(build_temp))
         with spawn_context():
             self.spawn([cmake, '-S', str(ext.source_dir), '-B', str(build_temp), *cmake_args])
             self.spawn([cmake, '--build', str(build_temp), *build_args])
